### PR TITLE
updates to migrate to labs.waterdata

### PR DESCRIPTION
**Pull request by dblodgett-usgs**
A lot of changes here, tests all pass locally now. We still need to get the draw schema into the new geoserver, so those end points have not changed.

I auto generated the README.md

### CODE REVIEW

File: tests/testthat/00_create_test_data.R [hunk 1]
```diff
@@ -20,13 +20,11 @@ test_that("test data can be created", {
   
   saveRDS(webprocess(DELIMITER = 'TAB'), "data/test_webprocess_tab.rds")
   
-  gconfig('wps.url'="https://cida.usgs.gov/gdp/process/WebProcessingService")
+  gconfig('wps.url'="https://labs.waterdata.usgs.gov/gdp-process-wps/WebProcessingService")
   wp <- webprocess()
   saveRDS(wp, "data/test_webprocess_knife_prod.rds")
   saveRDS(query(wp, 'algorithms'), "data/test_webprocess_algorithms_prod.rds")
   
-  # gconfig(wps.url = "https://cida-test.er.usgs.gov/gdp/process/WebProcessingService")
```
Comment (dblodgett-usgs) [inline tests/testthat/00_create_test_data.R:28]: Just removing these multi-tier tests, we don't intend to do active development on the GDP going forward so a test tier will be unlikely.

File: tests/testthat/data/test_keeptimes.xml [hunk 2]
```diff
@@ -0,0 +1,103 @@
+<wps:Execute service="WPS" version="1.0.0" xsi:schemaLocation="http://www.opengis.net/wps/1.0.0 http://schemas.opengis.net/wps/1.0.0/wpsExecute_request.xsd" xmlns:ogc="http://www.opengis.net/ogc" xmlns:ows="http://www.opengis.net/ows/1.1" xmlns:wps="http://www.opengis.net/wps/1.0.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
```
Comment (dblodgett-usgs) [inline tests/testthat/data/test_keeptimes.xml:1]: This file was generated from the cida-based process WPS. It was being referenced by URL but that URL will no longer be active when the cida-based GDP is shut down.

File: tests/testthat/data/test_notimesjob.xml [hunk 3]
```diff
@@ -0,0 +1,79 @@
+<wps:Execute service="WPS" version="1.0.0" xsi:schemaLocation="http://www.opengis.net/wps/1.0.0 http://schemas.opengis.net/wps/1.0.0/wpsExecute_request.xsd" xmlns:ogc="http://www.opengis.net/ogc" xmlns:ows="http://www.opengis.net/ows/1.1" xmlns:wps="http://www.opengis.net/wps/1.0.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
```
Comment (dblodgett-usgs) [inline tests/testthat/data/test_notimesjob.xml:1]: This is the same as the file above.

File: tests/testthat/test-webdata_object.R [hunk 4]
```diff
@@ -21,13 +21,13 @@ test_that("create webdata from geojob", {
...
   testthat::skip_on_cran()
-  noTimesJob <- geojob('https://cida.usgs.gov/gdp/process/request?id=b327be82-8bd5-4a7e-8fda-4288c1a6ef3d')
+  noTimesJob <- geojob("data/test_notimesjob.xml")
   wd <- webdata(noTimesJob)
   expect_equal(length(times(wd)), 2)
   expect_is(times(wd), "POSIXct")
   
   #make sure times aren't dropped
-  wd <- webdata(geojob(xml = 'https://cida.usgs.gov:443/gdp/process/request?id=dded6ef9-1e52-4fec-b14d-6a93606e7b1f'))
+  wd <- webdata(geojob(xml = "data/test_keeptimes.xml"))
```
Comment (dblodgett-usgs) [inline tests/testthat/test-webdata_object.R:30]: See two files added to the repo.

**Comment by wdwatkins** [comment]
Sorry I missed this—it looks good to me though.

**Pull request merged**